examples/pytorch/language-modeling/run_clm.py

```diff
@@ -279,6 +290,9 @@ def main():
     else:
         config = CONFIG_MAPPING[model_args.model_type]()
         logger.warning("You are instantiating a new config instance from scratch.")
+        if model_args.config_overrides is not None:
+            print("Overriding config:", model_args.config_overrides)
+            config.update(model_args.config_overrides)
```

Comment:
Here it would be easier to cast each value to the right type I think.

Reply:
I added a new method in configuration_utils - so now anybody can use it.